src/compare_vocabs.py

```diff
@@ -83,6 +91,11 @@ def main(vocab_dir: Path):
 
 
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Compare two versions of a list of vocabulary terms")
+    parser.add_argument("--old-terms", required=True, help="Path to the old vocabulary JSON file")
+    parser.add_argument("--new-terms", required=True, help="Path to the new vocabulary JSON file")
+
+    args = parser.parse_args()
     VOCAB_DIR = Path(__file__).parents[1] / "vocab"
 
-    main(vocab_dir=VOCAB_DIR)
+    main(vocab_dir=VOCAB_DIR, old_terms_path=Path(args.old_terms), new_terms_path=Path(args.new_terms))
```

Comment:
You haven't updated the output part of the script yet - i.e the `old_terms_unique.json` will always be whatever comparison you ran most recently

Reply:
Do you mean that it will override outputs from previous runs of compare_vocabs.py? I think that was intentional, since this script is meant to be more of a sanity check, but I've added an optional argument for output directory just in case :)